src/jobs/jobs.controller.ts

```
@@ -69,5 +98,6 @@ export class JobsController {
     // TODO: Get userId from request once authentication is implemented
     const userId = req.user?.id || '1'; // Placeholder
     return this.jobsService.isJobSaved(+id, userId);
+main
   }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)